coffee/chosen.native.coffee

```diff
@@ -0,0 +1,512 @@
+class @Chosen extends AbstractChosen
+
+  setup: ->
+    @current_selectedIndex = @form_field.selectedIndex
+    @is_rtl = DOM.has_class @form_field, "chosen-rtl"
+
+  set_up_html: ->
+    container_classes = ["chosen-container"]
+    container_classes.push "chosen-container-" + (if @is_multiple then "multi" else "single")
+    container_classes.push @form_field.className if @inherit_select_classes && @form_field.className
+    container_classes.push "chosen-rtl" if @is_rtl
+
+    container_props =
+      'class': container_classes.join ' '
+      'style': "width: #{this.container_width()};"
+      'title': @form_field.title
+
+    container_props.id = @form_field.id.replace(/[^\w]/g, '_') + "_chosen" if @form_field.id.length
+
+    @container = document.createElement 'div'
+    for key, value of container_props
+      @container.setAttribute key, value
+
+    if @is_multiple
+      @container.innerHTML = '<ul class="chosen-choices"><li class="search-field"><input type="text" value="' + @default_text + '" class="default" autocomplete="off" style="width:25px;" /></li></ul><div class="chosen-drop"><ul class="chosen-results"></ul></div>'
+    else
+      @container.innerHTML = '<a class="chosen-single chosen-default" tabindex="-1"><span>' + @default_text + '</span><div><b></b></div></a><div class="chosen-drop"><div class="chosen-search"><input type="text" autocomplete="off" /></div><ul class="chosen-results"></ul></div>'
+
+    @form_field.style.display = 'none'
+    @form_field.parentNode.appendChild @container
```

Comment:
This is not the same behavior as in the jquery or prototype version.

Should be:
```
@form_field.nextSibling ? @form_field.parentNode.insertBefore(@container, @form_field.nextSibling) : @form_field.parentNode.appendChild @container
```